engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -343,43 +345,24 @@ class WorkflowManagerActor(config: Config)
-      val startMode = if (isRestart) RestartExistingWorkflow else StartNewWorkflow
-      val wfActor = context.actorOf(ShadowWorkflowActor.props(workflowId, startMode, source), name = s"WorkflowActor-$workflowId")
-
-      wfActor ! SubscribeTransitionCallBack(self)
-      wfActor ! StartWorkflowCommand
-      logger.debug(s"Successfuly started ${wfActor.path} for Workflow ${workflowId}")
-      workflowId -> wfActor
-    }
-
-    def startOldeSchooleStylee = {
-      val message  = MaterializeWorkflowDescriptorActor.MaterializeWorkflow(workflowId, source, config)
-      val materializeWorkflowDescriptorActor = context.actorOf(MaterializeWorkflowDescriptorActor.props(), name = s"MaterializeWorkflowDescriptorActor-$workflowId")
-      materializeWorkflowDescriptorActor.askNoTimeout(message) map {
-        case MaterializeWorkflowDescriptorSuccess(descriptor) =>
-          val wfActor = context.actorOf(WorkflowActor.props(descriptor), name = s"WorkflowActor-$workflowId")
-          wfActor ! SubscribeTransitionCallBack(self)
-          wfActor ! (if (isRestart) Restart else Start(replyTo))
-          logger.debug(s"Successfuly started ${wfActor.path} for Workflow $workflowId")
-          context.stop(materializeWorkflowDescriptorActor)
-          workflowId -> wfActor
-        case MaterializeWorkflowDescriptorFailure(error) =>
-          val messageOrBlank = Option(error.getMessage).mkString
-          logger.error(error, s"$tag: Workflow failed submission: " + messageOrBlank)
-          replyTo foreach { _ ! WorkflowManagerSubmitFailure(error)}
-          // This error will be ignored currently as the only entity that needs to be notified about it
-          // has already been sent a failure message above
-          context.stop(materializeWorkflowDescriptorActor)
-          throw error
-      }
-    }
-
-    shadowExecutionEnabled match {
-      case true => Future.successful(startShadowStyle)
-      case false => startOldeSchooleStylee
+    val message  = MaterializeWorkflowDescriptorActor.MaterializeWorkflow(workflowId, source, config)
+    val materializeWorkflowDescriptorActor = context.actorOf(MaterializeWorkflowDescriptorActor.props(), name = s"MaterializeWorkflowDescriptorActor-$workflowId")
+    materializeWorkflowDescriptorActor.askNoTimeout(message) map {
+      case MaterializeWorkflowDescriptorSuccess(descriptor) =>
+        val wfActor = context.actorOf(WorkflowActor.props(descriptor), name = s"WorkflowActor-$workflowId")
+        wfActor ! SubscribeTransitionCallBack(self)
+        wfActor ! (if (isRestart) Restart else Start(replyTo))
+        logger.debug(s"Successfuly started ${wfActor.path} for Workflow $workflowId")
+        context.stop(materializeWorkflowDescriptorActor)
+        workflowId -> wfActor
+      case MaterializeWorkflowDescriptorFailure(error) =>
+        val messageOrBlank = Option(error.getMessage).mkString
+        logger.error(error, s"$tag: Workflow failed submission: " + messageOrBlank)
```

Comment:
This is legacy in that it only affects the non-shadow world. And it's not a change, except in removing it from a `{...}` block.

Unless you feel strongly otherwise, I intend to not action in this PR.